rattan-core/src/cells/mod.rs

```diff
@@ -78,10 +120,10 @@ impl Packet for StdPacket {
         }
     }
 
-    fn from_raw_buffer(buf: &[u8]) -> Self {
+    fn from_raw_buffer(buf: &[u8], timestamp: Instant) -> Self {
```

Comment:
I wonder if this is necessary, as I find that most of your real use cases just use Instant::now when calling this method. This is really a breaking change where I think is not necessary. I think we won't set this to another timestamp other than Instant::now in real usage, right? For test cases, you can just directly change this field's value.